### PR TITLE
Specifying supported Metrics for Embrace integration

### PR DESCRIPTION
**Pull request by scottieb3**
### What does this PR do?

Updating the metadata for Metrics supported in the Embrace integration

### Motivation

Following guidance from Datadog Tech Partnerships team

### Review checklist

- [ ✅ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ➖] Feature or bugfix has tests
- [ ] Git history is clean
- [ ➖ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ➖] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by scottieb3** [comment]
@davidlawrencer

**Comment by scottieb3** [comment]
@mfrishert

**Review by scottieb3 — COMMENTED** [review]
updated based on Datadog's response to open questions (via email)

**Comment by scottieb3** [comment]
@davidlawrencer + @mfrishert ( @jschoijet ): anything obvious I'm missing before I ping our Datadog contacts?  
It keeps failing on the csv, but I'm following docs ([here](https://docs.datadoghq.com/developers/integrations/check_references/#metrics-metadata-file) )

File: embrace_mobile/metadata.csv
Comment (eho1307) [inline embrace_mobile/metadata.csv:1]: Could you include descriptions as well?

**Review by eho1307 — CHANGES_REQUESTED** [review]

**Review by eho1307 — APPROVED** [review]

File: embrace_mobile/metadata.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+embrace.session_total.five_minute,count,300,,,5-minute count of sessions,0,embrace,sessions5m,
+embrace.session_total.hourly,count,3600,,,1-hour count of sessions,0,embrace,sessions1hr,
+embrace.session_total.daily,count,86400,,,1-day(UTC) count of sessions,0,embrace,sessions1d,
+embrace.crash_total.five_minute,count,300,,,5-minute count of crashes,0,embrace,crashes5m,
+embrace.crash_total.hourly,count,3600,,,1-hour count of sessions,0,embrace,crashes1hr,
+embrace.crash_total.daily,count,86400,,,1-day(UTC) count of sessions,0,embrace,crashes1d,
+embrace.users_total.daily,count,86400,,,1-day(UTC) count of unique users,0,embrace,users1d,
```
Comment (buraizu) [inline embrace_mobile/metadata.csv:8]:
```suggestion
embrace.session_total.daily,count,86400,,,1-day (UTC) count of sessions,0,embrace,sessions1d,
embrace.crash_total.five_minute,count,300,,,5-minute count of crashes,0,embrace,crashes5m,
embrace.crash_total.hourly,count,3600,,,1-hour count of sessions,0,embrace,crashes1hr,
embrace.crash_total.daily,count,86400,,,1-day (UTC) count of sessions,0,embrace,crashes1d,
embrace.users_total.daily,count,86400,,,1-day (UTC) count of unique users,0,embrace,users1d,
```

**Review by buraizu — APPROVED** [review]
Approving with a minor update requested for the metric descriptions

**Review by bgoldberg122 — CHANGES_REQUESTED** [review]
Left a minor comment, otherwise this should be good to go

**Review by bgoldberg122 — APPROVED** [review]

**Pull request merged**